okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/Platform.java

```diff
@@ -105,47 +119,71 @@ public void connectSocket(Socket socket, InetSocketAddress address,
   }
 
   /** Attempt to match the host runtime to a capable Platform implementation. */
+  @SuppressWarnings("unchecked")
   private static Platform findPlatform() {
-    // Attempt to find Android 2.3+ APIs.
+    // Find the conscrypt security provider if we can
+    Class<Provider> conscryptProviderClass = null;
     try {
+      conscryptProviderClass =
+          (Class<Provider>) Class.forName("org.conscrypt.OpenSSLProvider");
+    } catch (Throwable t) {
```

Comment:
If there is ever a LinkageError, I'd want my program to fail. Something would be very wrong with the build and I wouldn't want to silently have different providers used.